content_purge.vcl

```diff
@@ -0,0 +1,106 @@
+sub vcl_recv {
+
+    # invalidate:dydneecirnitnoikOowmucnaygAwjalp (testing cluster)
+    if (req.method ~ "(PURGE|BAN)") {
```

Comment:
We can also add an acl to whitelist IP who can purge a cache request (more secure)